classes/Http/Controller/ForgotController.php

```diff
@@ -121,10 +120,7 @@ public function resetAction(Request $request)
         }
 
         if ($error > 0) {
-            /** @var Session\Session $session */
-            $session = $this->service('session');
```

Comment:
Since we would otherwise be invoking `$request->getSession()` multiple times, can we maybe just use `$session` here and assign it `$request->getSession()` (we probably need to pull it out of the condition)?

Reply:
See the discussion above.